src/app/search/Preparation.ts

```
@@ -0,0 +1,344 @@
+
+export type DeepUnpreparedObject<T> = {
+    [K in keyof T as
+        K extends `${typeof SYMBOL_PREFIX}${string}` ? never
+        : T extends PreparedForMark<unknown>
+            ? K extends keyof Original<T> ? K
+            : never
+        : K
+    ]: DeepUnpreparedValue<T[K]>;
+} & (T extends PreparedForMark<infer U> ? {
+    [K in keyof U as K extends symbol ? K extends ReservedSymbol ? never : K : never]: DeepUnpreparedValue<DeepPreparedValue<U[K]>>;
+} : {})
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)